lib/Frontend/ArgsToFrontendOptionsConverter.cpp

```diff
@@ -498,6 +503,40 @@ bool ArgsToFrontendOptionsConverter::setUpImmediateArgs() {
   return false;
 }
 
+bool ArgsToFrontendOptionsConverter::computeModuleAliases() {
+  auto list = Args.getAllArgValues(options::OPT_module_alias);
+  if (!list.empty()) {
+    for (auto val: list) {
+      auto i = val.find("=");
```

Comment:
Tip: If you convert `val` to a `StringRef` right away, you can use `split("=")` here, which will avoid the need to fumble around with `substr` later. (If there's no `=` in the string, the second string in the pair will be empty.)